target/linux/generic/pending-5.10/484-mtd-spi-nor-add-esmt-f25l16pa.patch

```diff
@@ -0,0 +1,11 @@
+--- a/drivers/mtd/spi-nor/esmt.c
++++ b/drivers/mtd/spi-nor/esmt.c
+@@ -10,6 +10,8 @@
+ 
+ static const struct flash_info esmt_parts[] = {
+ 	/* ESMT */
++	{ "f25l16pa", INFO(0x8c2115, 0, 64 * 1024, 64,
```

Comment:
IIRC the number of sectors should be 32(I mean `64 * 1024, 32`), I've just noticed it.

Reply:
Oh, my mistake. I haven't noticed it until now... Thank you for letting me know about it!